src/ClientData/DataManagerTest.cpp

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gtest/gtest.h>
+
+#include <thread>
+#include <vector>
+
+#include "ClientData/DataManager.h"
+#include "OrbitBase/ThreadUtils.h"
+
+namespace orbit_client_protos {
+static bool operator==(const CallstackEvent& lhs, const CallstackEvent& rhs) {
+  return lhs.thread_id() == rhs.thread_id() && lhs.callstack_id() == rhs.callstack_id();
+}
+}  // namespace orbit_client_protos
+
+namespace orbit_client_data {
+
+using orbit_client_protos::CallstackEvent;
+using std::vector;
+
+TEST(DataManager, CallstackEventSelection) {
+  DataManager manager;
+  EXPECT_EQ(0, manager.GetSelectedCallstackEvents(orbit_base::kAllProcessThreadsTid).size());
+
+  const uint32_t t0_id = 0, t1_id = 1, e0_id = 0, e1_id = 1;
```

Comment:
Nit: Better if we follow google guidelines for constants.
 
go/cstyle#Constant_Names

Reply:
Oh right, I missed this... I'll change it in another PR.